docs/en/stack/ml/df-analytics/ml-dfa-outlier-detection.asciidoc

```diff
@@ -39,90 +33,130 @@ You can create {oldetection} {dfanalytics-jobs} in {kib} or by using the
-
-While the results of the distance based methods are easy to interpret, their 
-drawback is that they don't take into account the density variations of a 
-data set. This is the point where **density based methods** come into the 
-picture, they are used for mitigating this problem. These methods take into 
-account not only the distance of the points to their K nearest neighbors but 
-also the distance of these neighbors to their neighbors.
-
-//[role="screenshot"]
-//image::ml/images/ml-densitybm.jpg["Density based method – By Chire - Own work, Public Domain, https://commons.wikimedia.org/w/index.php?curid=10423954"]
-
-Based on this approach, a metric is computed called local outlier factor 
-(`lof`) for each data point. The higher the local outlier factor, the more 
-outlying is the data point.
-
-The other density based method that {oldetection} uses is the local 
-distance-based outlier factor (`ldof`). Ldof is a ratio of two measures: the 
+* distance of K^th^ nearest neighbor: computes the distance of the data point to 
+its K^th^ nearest neighbor where K is a small number and usually independent of 
+the total number of data points.
+* distance of K-nearest neighbors: calculates the average distance of the data 
+points to their nearest neighbors. Points with the largest average distance will 
+be the most outlying.
+* local outlier factor (`lof`): takes into account the distance of the points to 
+their K nearest neighbors and also the distance of these neighbors to their 
+neighbors.
+* local distance-based outlier factor (`ldof`): is a ratio of two measures: the 
 first computes the average distance of the data point to its K nearest 
 neighbors; the second computes the average of the pairwise distances of the 
-neighbors themselves. Again, the higher the value the more the data point is an 
-outlier.
+neighbors themselves.
+//end::outlier-detection-algorithms[]
 
-As you can see, these four algorithms work differently, so they don't always 
-agree on which points are outliers. By default, we use all these methods during 
-{oldetection}, then normalize and combine their results and give every datapoint 
-in the index an {olscore}. The {olscore} ranges from 0 to 1, where the higher 
-number represents the chance that the data point is an outlier compared to the 
-other data points in the index.
+You don't need to select the methods or provide any parameters, but you can 
+override the default behavior if you like. **Distance based methods** assume 
+that normal data points remain closer or similar in value while outliers are 
+located far away or significantly differ in value. The drawback of these methods 
+is that they don't take into account the density variations of a data set. 
+**Density based methods** are used for mitigating this problem.
+
+The four algorithms don't always agree on which points are outliers. By default, 
+we use all these methods during {oldetection}, then normalize and combine their 
```

Comment:
I generally try to avoid "we" if possible unless it's a hands-on tutorial kind of tone
```suggestion
{oldetection} jobs use all these methods, then normalize and combine their 
```